packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -124,8 +124,9 @@ Use a (Multiselect) [Combobox](/components/combobox) when:
 
 - You need a Twilio-styled options list.
 - You need to show more than text in an option (e.g., text paired with an icon).
-- You need placeholder text in the field to show an example option, rather than a preselected option.
-- Users would benefit from typeahead functionality (e.g., autocomplete, search). For example, typeahead may be useful when users need to select from a list of more than 10 options.
+- You need to group options under labels.
+- You need to disable options in the list.
+- Users would benefit from autocomplete functionality (e.g., autocomplete, search). For example, autocomplete may be useful when users need to select from a list of more than 10 options.
 - You need to lazy load a much longer list of options to improve page load performance.
 
 ### Accessibility
```

Comment:
GitHub won't let me add comments where there were no code changes, so I'm going to drop a list here:

1. Add an "Autocomplete Multi-select Combobox" example according to the spec here: https://docs.google.com/document/d/14BaGDkfoYddL8vFOQ-fH06AXicR7wu3PZNblW1U8ITs/edit#bookmark=id.agaz9x1c3vvv
2. The heading "Mutliselect Combobox with an empty state" should be "Multiselect Combobox with an empty state"
3. The `{emptyStateExample}` needs to show an empty state.

Reply:
> Add an "Autocomplete Multi-select Combobox" example according to the spec here: https://docs.google.com/document/d/14BaGDkfoYddL8vFOQ-fH06AXicR7wu3PZNblW1U8ITs/edit#bookmark=id.agaz9x1c3vvv

All Multiselects are autocomplete because it doesn't make sense otherwise, unlike regular Combobox, so a unique example isn't needed.

> The {emptyStateExample} needs to show an empty state.

It does if you search for something not in the list. I updated the helptext in that example for clarity.


Completed all the other items

Reply:
True, good points all around. I think the options section might've been c/p from Select docs.

And hrm, maybe the empty state example wasn't working for me only in the single-select. Might've been looking on the wrong page.

Thanks!